### ISSUE TITLE
Gauge metric to help debug LevelDB compactions?

### ISSUE THREAD

**Issue by Aracki** [ethereum/go-ethereum#27497]
# Rationale

We have only `eth_db_chaindata_compact_nonlevel0` & `eth_db_chaindata_compact_level0` regarding LevelDB compaction levels.

It would be helpful to have the gauge metric which would represent the current number of L0, L1, L2,... tables.
Now it can be monitored only within LevelDB logs. 

e.g.
```
17:32:48.421253 table@move L3@151145047 -> L4
17:32:48.501054 version@stat F·[25 380 3851 38027 101486 656155 3398169] S·7TiB[96GiB 768MiB 7GiB 75GiB 150GiB 976GiB 6TiB] Sc·[6.25 7.68 7.69 7.69 1.54 1.00 0.67]
```
```[tasklist]
### Tasks
```

**Comment by rjl493456442** [comment]
Reasonable request, would you like to submit a PR for it?

**Comment by Aracki** [comment]
I would be happy to help if I find time. It's good to hear there are reasons to add it. 🤞

**Comment by vvarl0cks** [comment]
let's see.

**Comment by rjl493456442** [comment]
It's merged in #27643
